QRHunter/app/src/main/java/com/example/qrhunter/ui/profile/ProfileViewModel.java

```diff
@@ -44,11 +49,21 @@ public LiveData<Player> getPlayer(String playerId) {
     public LiveData<ArrayList<QRCode>> getScannedQRCodes(Player player) {
         qrCodeRepository.getScannedQRCodes(player, scannedQRCodes -> {
             this.scannedQRCodes.setValue(scannedQRCodes);
+            this.highScore.setValue(PlayerUtil.calculateHighestScore(scannedQRCodes));
+            this.lowScore.setValue(PlayerUtil.calculateLowestScore(scannedQRCodes));
```

Comment:
My mistake, I think told u to put them here last time but they should be outside the `getScannedQRCodes` because these lines won't be called if the user has no scanned QR Codes

Reply:
Oh no problem, want me to make the changes and pull from main first and then push again?

Reply:
Nope, you pull from Feature/ranksredone (this branch) and then make your changes and then push it to Feature/ranksredone (this branch). You shuldn't be switching branches in this case